### DISCUSSION TITLE
Forecast against story points?

### DISCUSSION THREAD

**Discussion by ramsaytom** [nbrown02/FlowViz#73] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
A burn down chart related to effort? you have done an excellent forecast for completed items however do you think you will be able to create a forecast against effort/story points?

**Comment by nbrown02** [comment, marked as answer]
@ramsaytom - unfortunately not. I am not a fan of story points and believe the industry needs to move away from them :)

For more on my thoughts as to why this is - check out [this series of blogs](https://medium.com/nationwide-technology/story-pointless-part-1-of-3-a8fc3941691d) :)

**Reply by ramsaytom** [reply]
@nbrown02  I'm inclined to agree with, also great blog.

**Reply by nbrown02** [reply]
Thank you :)
